### DISCUSSION TITLE
Teslamate on Raspberry Pi 1 Model B

### DISCUSSION THREAD

**Discussion by vishnumrao** [teslamate-org/teslamate#3203] [Q&A]
I heard about teslamate on the TMC forum and I am very keen to try it. Using the instructions I was able to install docker and run the Hello world example on the Raspberry PI 1 Model B with with 512MB. Unfortunately I believe teslamate is not supported on the RPI1 due to its older arm generation.

Feature request:  would it be possible to add support for the RPI1.

I already have an RPI1 and with the shortage of RPI and the price gouging, its nearly impossible to find a new one. Hence the request.

**Comment by cwanja** [comment]
I doubt this will ever work due to the requirements of 1GB of RAM per the Docker installation documentation. https://docs.teslamate.org/docs/installation/docker

**Reply by vishnumrao** [reply]
Thanks for pointing that out. Maybe thats a good reason, but I am looking for a confirmation that RAM is indeed a limiting factor for not having a armv6 docker image.

I checked on a friends teslamate install on a RPI3 with 1GB RAM. Using the free command, it appears the entire RAM usage is only ~300MB with teslamate, pihole and owntone music server running. So its possible RAM may not be the limiting factor. I checked my own installation on the RPI1, it appears its using only ~50MB of DRAM as checked with the free command.

**Comment by ktsenvolt** [comment]
https://www.canakit.com/raspberry-pi-400-desktop-computer-kit.html

**Reply by vishnumrao** [reply]
Thanks. I am aware of official distributors for the RPI. But I did not initiate this thread to get buying advice. The goal was to make a feature request and if its not possible to implement feature request, I wanted to possibly understand the technical limitations that inhibit the request.
